mkosi/__init__.py

```diff
@@ -2170,6 +2173,14 @@ def run_verb(args: MkosiArgs, presets: Sequence[MkosiConfig]) -> None:
 
     last = presets[-1]
 
+    if args.verb == Verb.download:
+        workspace = tempfile.TemporaryDirectory(dir=last.workspace_dir, prefix=".mkosi-tmp")
```

Comment:
I'm necessarily using `last` here, for the sake of simplicity while testing. But from my understanding we can receive a list (or `Sequence`) of `MkosiConfig`.

Each config can carry it's own distribution. So perhaps we should, for each `preset`, download build cache of its distribution?

Reply:
Marking this as resolved since Daan clarified we should loop over all presets.